staging/src/k8s.io/kubectl/pkg/cmd/create/create_secret.go

```diff
@@ -370,25 +370,31 @@ func handleSecretFromFileSources(secret *corev1.Secret, fileSources []string) er
 	return nil
 }
 
-// handleSecretFromEnvFileSource adds the specified env file source information
```

Comment:
You could have kept that function and call it for each individual file? :-)

Reply:
That could be another way to do it! but would that need another change from the configmap file to keep things in sync?

Reply:
I think it's fine if you leave it like this. thanks